content/introduction/core-principles.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Core Principles"
```

Comment:
This feels light to me, though it's what we had written down. Let's put at least one or two more items into this doc to make sure it's well rounded.

Reply:
Agree - but let's just keep plowing through this. We can add more as they come up in office hours. I'll keep updating the Google Doc too. Probably will try to use suggest mode when I'm not on my phone.

Reply:
Sounds solid 👍